gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/driver/remote/RemoteWorld.java

```diff
@@ -159,19 +159,35 @@ public GraphTraversalSource getGraphTraversalSource(final LoadGraphWith.GraphDat
 
     ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-    public static class GraphBinaryRemoteWorld extends RemoteWorld {
-        public GraphBinaryRemoteWorld() { super(createTestCluster(SerializersV4.GRAPHBINARY_V4)); }
+    public static class GraphBinaryLangRemoteWorld extends RemoteWorld {
+        public GraphBinaryLangRemoteWorld() { super(createTestCluster(SerializersV4.GRAPHBINARY_V4)); }
 
         @Override
         public GraphTraversalSource getGraphTraversalSource(final LoadGraphWith.GraphData graphData) {
             final GraphTraversalSource g = super.getGraphTraversalSource(graphData);
-            return g.with("language", "gremlin-lang");
+             return g.with("language", "gremlin-lang");
```

Comment:
`with("language", "gremlin-lang")` is being used to rig up this test. i assume this isn't meant to be something users can do though? like, if they did `.with("language", "gremlin-groovy")` they would have a problem because you may or may not get groovy compliant output. maybe this is one of the request options we shouldn't allow with the language variants through `with()`? i dont' know what that means for testing through.......